src/lib/Components/Bidding/Elements/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components/native"
+import { borderColor, borders, color, flex, fontSize, height, space, width } from "styled-system"
+import { Fonts } from "../../../data/fonts"
+import {
+  BorderColorProps,
+  BorderProps,
+  ColorProps,
+  FlexboxProps,
+  FontSizeProps,
+  HeightProps,
+  SpaceProps,
+  WidthProps,
+} from "./types"
+
+export interface InputProps
+  extends BorderColorProps,
+    BorderProps,
+    ColorProps,
+    FlexboxProps,
+    FontSizeProps,
+    HeightProps,
+    SpaceProps,
+    WidthProps {}
+
+export const Input = styled.TextInput.attrs<InputProps>({})`
```

Comment:
Should we add a comment above this component, and the other `Input`, describing their purpose? I'm not clear on the difference now, between a component and an element.

Reply:
The responsibility of the components in the `elements` dir is to expose `styled-system`'s interface while the components in `components` dir can be opinionated about how we use them in the bidding flow. Right now, the `Grid` component is the only one that goes against this philosophy. We could move it to the `components` if we want to make the boundary more clear. We could also add a prefix to make them more self-documenting.